util/helpfultriggerwrappers.py

```diff
@@ -18,13 +18,14 @@ def __init__(self, axis: Axis, threshold: float) -> None:
         super().__init__(lambda: self.axis() > self.threshold)
 
 
-class SmartDashboardButton(Trigger):
+class NetworkTableButton(Trigger):
     def __init__(self, key: str) -> None:
-        self.key = NetworkTableInstance.getDefault().getBooleanTopic(key)
-        self.key.publish().set(False)
-
-        self.get = self.key.subscribe(False)
-        super().__init__(self.get.get)
+        super().__init__(
+            lambda: NetworkTableInstance.getDefault()
```

Comment:
i like the anon function usage lol, but you can just do `NetworkTableInstance.getDefault().getBooleanTopic(key).subscribe(False).get` which would be more efficient since every call doesn't re-get the topic value and subscribe to it, but rather just the getter. The purpose of the previous implementation was this memory saving as well.

I'd have the subscriber as a property of the class and then pass the get method (no parenthesis so you don't call it)